docker/base/centos7/Dockerfile

```diff
@@ -26,7 +27,10 @@ RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
     sudo \
     vim \
     wget \
-    which
+    which \
+    yum-utils \
+ && sed -i 's/http:\/\/vault.centos.org/https:\/\/vault.centos.org/g' /etc/yum.repos.d/* \
+ && yum upgrade ca-certificates
```

Comment:
Why this change?

Reply:
Ahh Good catch! Forgot to mention this in the summary. 

It looks like the centos7 yum repos are now requiring https connections and not allowing http, which is the default for centos7. This seems like the easiest way to make that change. The `sed` command changes the address to use https, and the `yum upgrade` is required to have the certificates to use the https connection.